src/edu/stanford/nlp/patterns/TextAnnotationPatterns.java

```diff
@@ -41,10 +41,6 @@
   private static final Redwood.RedwoodChannels logger = Redwood.channels(TextAnnotationPatterns.class);
 
   public TextAnnotationPatterns() throws IOException {
```

Comment:
I think the commented code is here so that someone working on this has a convenient suggestion on how to log intermediate results, but I could be wrong about this